From 6ed9946f9a4f1a01846add2279e8d0640c1c2f1c Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Thu, 6 May 2021 12:53:58 +0200
Subject: tools: hv: change http to https in hv_kvp_daemon.c

The patch has no functional change. Only changes the URL pointed to
in one of the comments

commit fa52a4b2d0ab416508538bb47a95167d4c94caac
Author: Alexander A. Klimov <grandmaster@al2klimov.de>
Date:   Sun Jul 5 23:44:57 2020 +0200

    tools: hv: change http to https in hv_kvp_daemon.c

    Rationale:
    Reduces attack surface on kernel devs opening the links for MITM
    as HTTPS traffic is much harder to manipulate.

    Deterministic algorithm:
    For each file:
      If not .svg:
        For each line:
          If doesn't contain `\bxmlns\b`:
            For each link, `\bhttp://[^# \t\r\n]*(?:\w|/)`:
              If both the HTTP and HTTPS versions
              return 200 OK and serve the same content:
                Replace HTTP with HTTPS.

    Signed-off-by: Alexander A. Klimov <grandmaster@al2klimov.de>
    Link: https://lore.kernel.org/r/20200705214457.28433-1-grandmaster@al2klimov.de
    [ wei: change subject line to be more specific ]
    Signed-off-by: Wei Liu <wei.liu@kernel.org>

Signed-off-by: Mohammed Gamal <mgamal@redhat.com>
(cherry-picked from rhel 8.4.0 commit e956573)
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hv_kvp_daemon.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hv_kvp_daemon.c b/hv_kvp_daemon.c
index 0e5f14a..c06c94d 100644
--- a/hv_kvp_daemon.c
+++ b/hv_kvp_daemon.c
@@ -437,7 +437,7 @@ void kvp_get_os_info(void)
 
 	/*
 	 * Parse the /etc/os-release file if present:
-	 * http://www.freedesktop.org/software/systemd/man/os-release.html
+	 * https://www.freedesktop.org/software/systemd/man/os-release.html
 	 */
 	file = fopen("/etc/os-release", "r");
 	if (file != NULL) {
-- 
2.27.0

